tests/unit/tests_suite_test.go

```diff
@@ -16,6 +20,43 @@ const (
 	helmReleaseName             = "k8ssandra-test"
 )
 
+// Common labels for k8ssandra chart
+func GetK8ssandraRequiredLabels() map[string]string {
```

Comment:
Just a heads up. The k8ssandra-common chart defines helper templates that we want to use across the charts. This needs to be done as part of the chart refactoring that is a WIP. See [here](https://github.com/k8ssandra/k8ssandra/blob/main/charts/cass-operator/templates/deployment.yaml#L5) for an example.

Reply:
I suspect we would want to have the unit-level test "base" for the common do that work?  

Is there something I can reference in the WIP or is this post-1.0?  I haven't been tracking that effort.

Reply:
@jsanda below is the set of labels collected as what I understand to be the `common-labels` standard.
```
	helm.sh/chart:                "k8ssandra-" + $meta.Version,
	app.kubernetes.io/name:       "k8ssandra",
	app.kubernetes.io/instance:   $helmReleaseName,
	app.kubernetes.io/version:    $meta.AppVersion,
	app.kubernetes.io/managed-by: "Helm"
```	
It appears `k8ssandra-common-labels` is the `common-labels` + the label `app.kubernetes.io/part-of`.

```
{{- define "k8ssandra-common.labels" }}
{{ include "common.labels.standard" . }}
app.kubernetes.io/part-of: ": 	helmReleaseName+"/"+defaultTestNamespace,
```

**Questions:**

1. `k8ssandra-common/templates/_helpers.tpl` has the k8ssandra-common.labels commented out.  Is this where we want to define just the common-labels set?
2. For the `k8ssandra/templates/_helpers.tpl`, is this where we want to define our k8ssandra-common.labels?
3. From a unit test perspective, does it make sense to reference the `common-labels` and the `k8ssandra-common-labels` to verify that select templates include these labels?